models/media_ad_views/media_ad_views.yml

```diff
@@ -0,0 +1,57 @@
+version: 2
+
+models:
+  - name: snowplow_media_player_media_ad_views
```

Comment:
I know this may require a bigger rework due to the integration tests but wouldn't it make sense to add the `session_identifier` to the `media_ad_views` and `media_ads`? Was just thinking if the user journey needs to be analysed (thinking of similar models like the custom session stats) it would be easier to just join directly to other session based calculations rather than having to first join the base through the play_id to get the session_identifier? If it's a lot we could add later I guess, don't want to prolong the PR forever, it's rather a nice to have.

Reply:
That makes sense, I have added `session_identifier` to `media_ad_views`! I don't think we can add it to `media_ads` because that aggregates for the whole ad, doesn't report on the level of sessions.

Reply:
thanks!!